Jenkinsfile

```diff
@@ -109,7 +109,7 @@ pipeline {
                             filename 'Dockerfile.centos:7'
                             dir 'utils/docker'
                             label 'docker_runner'
-                            additionalBuildArgs "-t ${sanitized_JOB_NAME}-centos7 " + '$BUILDARGS'
+                            additionalBuildArgs "-t ${sanitized_JOB_NAME}-centos7 " + '$BUILDARGS --build-arg DONT_USE_RPMS=false --build-arg OMPI=ompi-3.0.0rc4-1'
```

Comment:
In general I find variable names that contain a negative to not be a good idea, although this is not a regression.

Reply:
I generally avoid them myself, but sometimes they are more useful as a double negative.  I will review that in this case the next time I am looking over this patch.